src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -1222,14 +1222,14 @@ public override BoundNode VisitSequence(BoundSequence node)
 
             var sideEffects = VisitExpressionList(ref builder, node.SideEffects, forceSpill: valueBuilder != null, sideEffectsOnly: true);
 
-            if (builder == null && valueBuilder == null)
+            if (builder == null && valueBuilder == null && !node.ForceSpill)
```

Comment:
>!node.ForceSpill [](http://example.com/codeflow?start=59&length=16)

It doesn't look like the meaning of this flag is consistent with what this component is using for the similar term. For example, in the ```VisitExpressionList``` call above. If I remember correctly, in there "forceSpill" actually forces the spilling, i.e. a call of the ```Spill``` helper. It appears to me that what the new flag is doing, it forces us to "unwrap" the sequence rather than actually spill its value.